File: 05. Lenguajes no clasificados/02. Visual Basic/18. Matrices/Matrices.vb
Module Module1

    Sub Main()

        ' Matriz dinámica
        Dim arreglo1(1, 1) As Integer
        arreglo1(0, 0) = 234
        arreglo1(0, 1) = 235
        arreglo1(1, 0) = 236
        arreglo1(1, 1) = 237

        ' Matriz estática
        Dim arreglo2(,) As Integer = {{234, 235}, {236, 237}}

        ' Imprimir matriz
        For i As Integer = 0 To arreglo2.GetLength(0) - 1 Step 1
            For j As Integer = 0 To arreglo2.GetLength(1) - 1 Step 1
                Console.WriteLine(arreglo2(i, j))
            Next
        Next

        Console.ReadLine()

    End Sub

End Module

File: 05. Lenguajes no clasificados/01. C#/18. Matrices/Matrices.java
package prueba;

public class Prueba {

    public static void main(String[] args) {

        // Arreglo dinámico
        int[][] arreglo1 = new int[2][2];
        arreglo1[0][0] = 234;
        arreglo1[0][1] = 235;
        arreglo1[1][0] = 236;
        arreglo1[1][1] = 237;

        // Arreglo estático
        int[][] arreglo2 = {{234, 235}, {236, 237}};

        // Imprimir arreglo
        for (int i = 0; i < arreglo1.length; i++) {
            for (int j = 0; j < arreglo1[i].length; j++) {
                System.out.println(arreglo1[i][j]);
            }
            
        }

    }

}
